webpack.config.ts

```diff
@@ -177,4 +178,4 @@ const clientConfigProduction = {
 
 // ----- Exports ----- //
 
-module.exports = [ serverConfig, clientConfig, clientConfigProduction ];
+export default [ serverConfig, clientConfig, clientConfigProduction ];
```

Comment:
Not sure we need this export but kept it just in case I'm not aware of something.